web/styles/settings.css

```diff
@@ -513,11 +513,15 @@ input[type="checkbox"] {
     vertical-align: top;
     height: auto !important;
     width: auto !important;
+    white-space: break-spaces !important;
 
+    word-break: normal;
```

Comment:
I think we can remove this since default for `word-break` is `normal`.

Reply:
made the change.